.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: go
 
 sudo: true
+dist: xenial
```

Comment:
could you explain this change?

Reply:
By default Travis uses Ubuntu Trusty, that have a very old version of Python 2.7, the 2.7.6. This version does not offer an easy way to skip the ssl check required to use the polling on Pebble directory url I add in this PR. 

Xenial ships with Python 2.7.11, that have this support.